x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/sequence/CircuitBreakerTests.java

```diff
@@ -306,6 +223,107 @@ private void assertMemoryCleared(int sequenceFiltersCount, BiFunction<CircuitBre
         }
     }
 
+    public void testParentCircuitBreakerOnClean() {
+        int sequenceFiltersCount = 2;
+        final int SEARCH_REQUESTS_EXPECTED_COUNT = 2;
+        List<BreakerSettings> eqlBreakerSettings = Collections.singletonList(
```

Comment:
The `eqlBreakerSettings` can be extracted in a separate method. This code block is used in two places in this class.